### PR TITLE
Enable Python 3.8 on Azure Pipelines

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Related to #3835

### CODE REVIEW

**Comment by aldanor** [comment]
Shouldn't lief / py-lief be built for py38 first? They seem to only exist for py37 atm.

**Comment by marcelotrevisani** [comment]
> Shouldn't lief / py-lief be built for py38 first? They seem to only exist for py37 atm.

Yeap, you are right! We should build it for py38 first!

cc: @msarahan

**Comment by mingwandroid** [comment]
Unfortunately LIEF doesn't work with Windows Python 3.8 yet and I'm not sure why. I'm working on enabling debug Python interpreters to be built with the python-feedstock recipe at present so I can debug this (and things like it) more easily.

**Comment by aldanor** [comment]
@mingwandroid Any further ideas on this?

(especially now that conda's py38 builds are out, this becomes the stumbling block...)

**Comment by mingwandroid** [comment]
Yup, thanks all!

**Comment by aldanor** [comment]
Great stuff, thanks! (would the py38 builds appear in the default channel at some point?)

File: azure-pipelines.yml [hunk 1]
```diff
@@ -21,6 +21,12 @@ jobs:
       py37-release:
         PYTHON_VERSION: '3.7'
         CONDA_VERSION: 'canary'
+      py38-canary:
+        PYTHON_VERSION: '3.8'
+        CONDA_VERSION: 'release'
+      py38-release:
+        PYTHON_VERSION: '3.8'
+        CONDA_VERSION: 'canary'
```
Comment (mrakitin) [inline azure-pipelines.yml:29]: Is not it swapped? The section name does not match the conda version...

**Pull request merged**